frontend/simple/views/Invite.vue

```diff
@@ -110,8 +116,12 @@ export default {
           contract.constructor.vuex.mutations[type](state, action.data)
         })
         let mailbox = await backend.latestHash(state.attributes.mailbox)
-        let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId}, mailbox)
+        let date = new Date()
+        let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId, sentDate: date.toString()}, mailbox)
```

Comment:
If dates always require a timestamp, then those should be generated by the constructor itself in `shared/events.js`. It's a DRY violation otherwise.